mmcv/runner/utils.py

```diff
@@ -13,7 +14,18 @@
 
 
 def get_host_info():
-    return f'{getuser()}@{gethostname()}'
+    """Get hostname and username.
+
+    Return empty string if exception raised, e.g. `getuser()` will lead to
+    error in docker container
+    """
+    host = ''
+    try:
+        host = f'{getuser()}@{gethostname()}'
+    except Exception as e:
+        warnings.warn('Host or user not found: %s' % (str(e)))
```

Comment:
We usually use f-string, so please change it to `f'Host or user not found: {str(e)}'`